docs/content/documentation/getting-started/overview.md

```diff
@@ -88,7 +88,7 @@ Let's make a template for a home page. Create `templates/base.html` with the fol
 <body>
   <section class="section">
     <div class="container">
-      {% block content %} {% endblock %}
+      {% block content %} {% endblock content %}
```

Comment:
It's not a type, the name of the block in endblock is optional

Reply:
I see, I saw the pattern in the [`Tera`](https://keats.github.io/tera/docs/#getting-started) docs and throughout the rest of the getting started, so I thought I'd throw up a PR. Feel free to close, if it's not needed.